src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -209,12 +213,27 @@ public static ImmutableArray<SyntaxNode> GetGetAccessorStatements(
             return [generator.ReturnStatement(expression)];
         }
 
-        return preferAutoProperties ? default : generator.CreateThrowNotImplementedStatementBlock(compilation);
+        if (preferAutoProperties)
```

Comment:
>if (preferAutoProperties) [](http://example.com/codeflow?start=8&length=25)

we don't want to do this for auto-properties when there are conflicting properties too?

Reply:
honestly.  i'm really trying to hit the sweet spot here.